src/Broadway/Saga/State/InMemoryRepository.php

```diff
@@ -16,7 +16,7 @@ class InMemoryRepository implements RepositoryInterface
     /**
      * {@inheritDoc}
      */
-    public function findBy(Criteria $criteria, $sagaId)
+    public function findBy(Criteria $criteria, $sagaId, $excludeRemoved = true)
```

Comment:
Zoals besproken kijken of er een mogelijkheid is om de excludeRemoved parameter mee in de Criteria te verwerken.